### PR TITLE
readme by jeelani

### PR DESCRIPTION
**Pull request by djeelani**
us/1234 to be checked-in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Skill-Portal
\ No newline at end of file
+# Skill-Portal
+
+Added by Jeelani
```
Comment (vbalas) [inline README.md]: Nice jeelani

Comment (vbalas) [inline README.md]: Reviewed

**Review by vbalas — APPROVED** [review]

**Comment by mpramodkumar** [comment]
pls resolve conflits

**Comment by djeelani** [comment]
done merge and commit..

**Pull request merged**